### PR TITLE
Added IsAiming and HasFlashlightModuleEnabled

### PR DESCRIPTION
**Pull request by Michal78900**

### CODE REVIEW

File: Exiled.API/Features/Player.cs [hunk 1]
```diff
@@ -392,10 +393,15 @@ public RoleType Role
         /// </summary>
         public bool IsReloading => CurrentItem is Firearm firearm && !firearm.Base.AmmoManagerModule.Standby;
 
-        /*/// <summary>
-        /// Gets a value indicating whether or not the player is zooming with a weapon.
+        /// <summary>
+        /// Gets a value indicating whether or not the player is aiming with a weapon.
+        /// </summary>
+        public bool IsAimingWeapon => CurrentItem is Firearm firearm && firearm.Base.AdsModule.ServerAds;
```
Comment (joker-119) [inline Exiled.API/Features/Player.cs:399]: We should probably make this a property in the Firearm class, then here do 
```cs
public bool IsAimingDownWeapon => CurrentItem is Firearm firearm && firearm.Aiming```

Comment (Michal78900) [reply Exiled.API/Features/Player.cs:399]: ok

File: Exiled.API/Features/Player.cs [hunk 2]
```diff
@@ -392,10 +392,15 @@ public RoleType Role
...
 
-        /*/// <summary>
-        /// Gets a value indicating whether or not the player is zooming with a weapon.
+        /// <summary>
+        /// Gets a value indicating whether or not the player is aiming with a weapon.
+        /// </summary>
+        public bool IsAimingDownWeapon => CurrentItem is Firearm firearm && firearm.Aiming;
+
+        /// <summary>
+        /// Gets a value indicating whether or not the player has enabled weapon's flashlight module.
```
Comment (PintTheDragon) [inline Exiled.API/Features/Player.cs:401]: Redundant "or not".

File: Exiled.API/Features/Player.cs [hunk 3]
```diff
@@ -392,10 +392,15 @@ public RoleType Role
         /// </summary>
         public bool IsReloading => CurrentItem is Firearm firearm && !firearm.Base.AmmoManagerModule.Standby;
 
-        /*/// <summary>
-        /// Gets a value indicating whether or not the player is zooming with a weapon.
+        /// <summary>
+        /// Gets a value indicating whether or not the player is aiming with a weapon.
```
Comment (PintTheDragon) [inline Exiled.API/Features/Player.cs:396]: Redundant "or not".

File: Exiled.API/Features/Items/Firearm.cs [hunk 4]
```diff
@@ -94,6 +91,16 @@ public byte Ammo
         /// </summary>
         public DamageTypes.DamageType DamageType => Base.DamageType;
 
+        /// <summary>
+        /// Gets a value indicating whether the Firearm is being aimed or not.
```
Comment (PintTheDragon) [inline Exiled.API/Features/Items/Firearm.cs:95]: Redundant "or not".

File: Exiled.API/Features/Items/Firearm.cs [hunk 5]
```diff
@@ -94,6 +91,16 @@ public byte Ammo
...
+        /// </summary>
+        public bool Aiming => Base.AdsModule.ServerAds;
+
+        /// <summary>
+        /// Gets a value indicating whether the Firearm's flashlight module is enabled or not.
```
Comment (PintTheDragon) [inline Exiled.API/Features/Items/Firearm.cs:100]: Redundant "or not".

**Review by PintTheDragon — CHANGES_REQUESTED** [review]
Looks good, just need to fix a few things in the docs.

Also,  I didn't comment this, but there's some inconsistent capitalization with "firearm". Sometimes it appears as "Firearm", sometimes as "firearm".

**Pull request merged**